manifests/version.pp

```diff
@@ -69,13 +78,20 @@
 
     rbenv::rehash { $version: }
 
-    # Save time and inodes by not installing gem documentation by default.
-    file { "${rbenv::params::rbenv_root}/versions/${version}/etc":
-      ensure => directory,
-    }
-    file { "${rbenv::params::rbenv_root}/versions/${version}/etc/gemrc":
-      ensure  => present,
-      content => "gem: --no-document --no-rdoc --no-ri\n",
+    if $install_gem_docs == 'yes' {
```

Comment:
Is this correct? If we're specifying `install_gem_docs` as `true` then we're ensuring this is absent?

Reply:
I think this is correct.

To disable gem documentation this file needs to be present (with the `--no-document` flags). To re-enable again, you'd need to remove this file.

Maybe there's a better / clearer way to do this though?

Reply:
👍